demo/public/app/cases/editors/DateEditor.js

```diff
@@ -9,7 +9,8 @@ define([ 'comindware/core', 'demoPage/views/EditorCanvasView' ], function (core,
             editor: new core.form.editors.DateEditor({
                 model: model,
                 key: 'dateValue',
-                autocommit: true
+                autocommit: true,
+                dateDisplayFormat: null

```

Comment:
These *displayFormat options should be removed here, as well as on other demo pages. The pages reflect the normal usage of the editor and, in normal scenario, the user shouldn't pass option `dateDisplayFormat: null` to make the editor work.